metrics-and-logs/index.html.markerb

```diff
@@ -0,0 +1,14 @@
+---
+title: "Metrics and Logs"
+layout: docs
+nav: firecracker
+toc: false
+---
+
+Fly.io metrics and logs.
+
+- **[Metrics](/docs/metrics-and-logs/metrics):** The fully-managed metrics that you can access for your app on Fly.io.
+- **[Logging overview](/docs/metrics-and-logs/logs/):** How logs work on Fly.io.
+- **[Live tail logs](/docs/metrics-and-logs/live-tail-logs/):** Where to access live tail logs.
+- **[Search logs](/docs/metrics-and-logs/search-logs/):** Experimental searchable app logs in Grafana. 
```

Comment:
Perhaps avoid saying 'experimental' here, as we want people to use this and think about it as an upcoming product. Maybe use 'beta' or nothing at all.